brianlong/www.validators.app
# 187182791 add edgevana stake pool

#### What's this PR do?
- Adds edgevana stake pool

#### How should this be manually tested?
- Only on localhost: if you still have some eversol stake pools, you'll need to clean them up first:
  - run `rails r script/one_time_scripts/remove_stake_pool.rb 'Eversol' 'mainnet'`
  - then run `rails r script/update_validator_stake_pools_list.rb`
- Run tasks: `rails add_stake_pools:mainnet` and `rails update_fees_in_stake_pools:mainnet`
- Create current epoch `rails r script/add_current_epoch.rb`, and then run: `rails r script/gather_stake_accounts.rb` 
- Review results in /stake-pools page
- Run `rails r script/update_validator_stake_pools_list.rb` and review the result on validators index and details pages, eg. this validator http://localhost:3000/validators/8MZFWowte8pHb6zJXxUdogUfdWRbqu7fGvaWRXtkDP7X?network=mainnet should have edgevana icon

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187182791)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
